doc/code_snippets/snippets/config/instances.enabled/flightrec/config.yaml

```diff
@@ -0,0 +1,11 @@
+groups:
+  group001:
+    replicasets:
+      replicaset001:
+        instances:
+          instance001:
+            iproto:
+              listen:
+              - uri: '127.0.0.1:3301'
+              flightrec:
+                enabled: true
```

Comment:
I'd expand this sample with a couple of configuration options and included it to the topic.